source/features/merged-pr-first-published-tag.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import TagIcon from 'octicon/tag.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../libs/features';
+import fetchDom from '../libs/fetch-dom';
+import {getRepoURL} from '../libs/utils';
+import observeElement from '../libs/simplified-element-observer';
+
+const getFirstTag = cache.function(async (commit: string): Promise<[string, string] | undefined> => {
+	const firstTag = await fetchDom<HTMLAnchorElement>(`/${getRepoURL()}/branch_commits/${commit}`, 'ul.branches-tag-list li:last-child a');
+	if (!firstTag) {
+		return;
+	}
+
+	return [firstTag.href, firstTag.textContent!];
```

Comment:
I wonder if we need both. I think you can easily generate the URL from the name
```suggestion
	return firstAssociatedTag?.textContent;
```